tests/testthat/test-standalone.R

```diff
@@ -18,9 +18,9 @@ test_that("We can compile the standalone program", {
   expect_equal(code, 0)
 
   output <- tempfile()
-  input <- frogger_file("fit_model/data")
+  input <- frogger_file("standalone_model/data")
```

Comment:
Trying to understand the nuance between standalone and simulate, is standalone the name of the input object and simulate is what you will call the function?

Reply:
Yes opportunistic renaming here. Standalone is just used as a directory name, so only a name used for organising, not a function or an object in the code. This just refers to the directory where the I've written the example of running the model entirely in C++, with no R data or wrapper involved. So it is standalone in that it is standalone C++.